connector_prestashop_feature/models/product_template/importer.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+import logging
+from openerp.addons.connector.unit.mapper import (
+    mapping,
+)
+from openerp.addons.connector_prestashop.backend import prestashop
+from openerp.addons.connector_prestashop.models.product_template import \
+    importer
+
+_logger = logging.getLogger(__name__)
+
+
+@prestashop(replacing=importer.FeaturesProductImportMapper)
+class FeaturesImportMapper(importer.FeaturesProductImportMapper):
+
+    @mapping
+    def extras_features(self, record):
+        custom_info_template = self.env.ref(
+            'connector_prestashop_feature.tpl_prestashop_features')
+        return {'custom_info_template_id': custom_info_template.id}
+
+
+@prestashop(replacing=importer.ProductTemplateImporter)
```

Comment:
This will be an issue the day we'll have 2 addons that try to replace importer.ProductTemplateImporter.